### PR TITLE
Add initial framebuffer debug tool

### PR DESCRIPTION
**Pull request by drebrez**
I've started with a basic debug tool that performs the `FBIOGET_FSCREENINFO` and `FBIOGET_VSCREENINFO` IOCTLs calls and print out all the values in the struct retrieved.

I've chose the name `fbdebug` to follow the busybox naming for framebuffer tools (`fbset`, `fbsplash`,...)

Please review and test it, thanks.

#1188

### CODE REVIEW

File: aports/main/fbdebug/fbdebug.c [hunk 1]
```diff
@@ -0,0 +1,160 @@
+/*
+Copyright (C) 2017 drebrez <drebrez@gmail.com>
```
Comment (ollieparanoid) [inline aports/main/fbdebug/fbdebug.c]: We have 2018 now :smiley:

File: aports/main/fbdebug/fbdebug.c [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+    printf("\treserved[0]      = %u\n", scr_var->reserved[0]);
+    printf("\treserved[1]      = %u\n", scr_var->reserved[1]);
+    printf("\treserved[2]      = %u\n", scr_var->reserved[2]);
+    printf("\treserved[3]      = %u\n", scr_var->reserved[3]);
+}
+
+int main(int argc, char** argv)
+{
+    const char *fb_device;
+	struct fb_fix_screeninfo scr_fix;
```
Comment (ollieparanoid) [inline aports/main/fbdebug/fbdebug.c]: Tabs and spaces mixed, I'd recommend to decide on one throughout the file.

**Comment by ollieparanoid** [comment]
It might make sense to create a wiki page, that explains how to use the tool (how you can use it to fix your framebuffer drivers).

File: aports/main/fbdebug/fbdebug.c (hunk 2 above)
Comment (drebrez) [reply aports/main/fbdebug/fbdebug.c]: thanks, I will fix it.
I wrote it using `vi` in the virtualmachine with alpine linux, I'm not so expert with `vi`.
I'll look for some way to display the special characters next time I have to use `vi`

**Comment by drebrez** [comment]
@ollieparanoid I think we can explain the various tools that we will add to the debug-shell hook once we have them.
Next week I will probably extend this tool to add additional tests, like the ioctl pan and draw something on the screen. I was thinking about something like a simplified version of a TV test pattern 😄

**Comment by eMPee584** [comment]
🤔 hmm why is it not called fbinfo 😃

**Comment by drebrez** [comment]
@eMPee584 `fbinfo` sounds more like a tool to only get information about it, the idea is to add additional functionalities to test various IOCTLs and writing into it.
Maybe we can call it `fbtest`? @ollieparanoid what you prefer?

**Comment by eMPee584** [comment]
maybe put the info part in a fbinfo executable, and the testing in a fbtest : )
or, rather go wild 🤠

**Comment by drebrez** [comment]
As you prefer, my plan is to add more features next week, but I can do it
in another PR

**Comment by ollieparanoid** [comment]
Alright, merged it. Smaller diffs are easier to review, and who knows, maybe it could already be useful :+1:

**Pull request merged**